student_class.rb

```diff
@@ -0,0 +1,11 @@
+require './person_class'
+class Student < Person
+  def initialize(classroom)
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Here at the time of initialization we have to pass four parameters. (age, classroom, name = "Unknown", parent_permission = true). some of them have default values. Kindly use-
```ruby
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```